spec/compiler/formatter/formatter_spec.cr

```diff
@@ -886,6 +886,219 @@ describe Crystal::Formatter do
       CRYSTAL
   end
 
+  context "adds trailing comma to def multi-line normal, splat, and double splat parameters" do
+    assert_format <<-CRYSTAL, <<-CRYSTAL, flags: %w[def_trailing_comma]
+      macro foo(
+        a,
+        b
+      )
+      end
+      CRYSTAL
+      macro foo(
+        a,
+        b,
+      )
+      end
+      CRYSTAL
+
+    assert_format <<-CRYSTAL, <<-CRYSTAL, flags: %w[def_trailing_comma]
+      macro foo(
+        a,
+        *b
+      )
+      end
+      CRYSTAL
+      macro foo(
+        a,
+        *b,
+      )
+      end
+      CRYSTAL
+
+    assert_format <<-CRYSTAL, <<-CRYSTAL, flags: %w[def_trailing_comma]
```

Comment:
missing a spec for varargs (`...`)

Reply:
varargs would need a parser update to support. I added a spec to ensure that it remains unchanged until if/when that other change is made.